kpm/lib/kpm/killbill_server_artifact.rb

```diff
@@ -61,14 +61,16 @@ def info(version = 'LATEST', sha1_file = nil, force_download = false, verify_sha
                               verify_sha1,
                               overrides,
                               ssl_verify)
-
-          pom = REXML::Document.new(File.new(oss_pom_info[:file_path]))
+          file = File.new(oss_pom_info[:file_path])
+          pom = REXML::Document.new(file)
           properties_element = pom.root.elements['properties']
           %w[killbill-api killbill-plugin-api killbill-commons killbill-platform].each do |property|
             versions[property] = properties_element.elements["#{property}.version"].text
+            file.close
           end
 
-          sha1_checker.cache_killbill_info(version, versions) if sha1_checker
+         sha1_checker.cache_killbill_info(version, versions) if sha1_checker
```

Comment:
Wrong indentation: this is making CI fail (https://app.circleci.com/pipelines/github/killbill/killbill-cloud/116/workflows/5fcc05c0-abd2-4d94-8af4-cfe89dbd45e0/jobs/3127).

You can run RuboCop locally by running `bundle exec rubocop -A` to fix these problems automatically.